src/AMP.php

```diff
@@ -49,7 +49,9 @@ class AMP
     // The StandardFixPass should be after StandardScanPass
     public $passes = [
         'Lullabot\AMP\Pass\PreliminaryPass', // Removes user blacklisted tags
-        'Lullabot\AMP\Pass\SugarImgTagTransformPass',
+        'Lullabot\AMP\Pass\ImgTagTransformPass',
```

Comment:
Ah, ok, makes sense. Was not obvious, since there is still file SugarStandardFixPass.